fuzz: Print error message when FUZZ is missing

maflcko opened the pull request:
Some trivial UX improvements.

* Change the exit code for `PRINT_ALL_FUZZ_TARGETS_AND_ABORT` and `WRITE_ALL_FUZZ_TARGETS_AND_ABORT` to `EXIT_SUCCESS` instead of `Aborted (core dumped)`.
* Print readable error message when `FUZZ` is missing instead of `Aborted (core dumped)`.
* Clarify that a fuzz target needs to be compiled into the executable.

Commit:
fuzz: Print error message when FUZZ is missing

Also, add missing includes.

dergoegge commented on src/test/fuzz/fuzz.cpp:
```diff
@@ -92,13 +97,22 @@ void initialize()
+        static std::string g_copy{env_fuzz}; // create copy to avoid compiler optimizations, and
+        g_fuzz_target = g_copy.c_str();      // strip string after the first null-char.
+    } else {
+        std::cerr << "Must select fuzz target with the FUZZ env var." << std::endl;
```
nit: Would it be good to hint at `PRINT_ALL_FUZZ_TARGETS_AND_ABORT=1` in the message?

dergoegge reviewed (commented):
Concept ACK

Commit:
ci: Log qa-assets repo last commit

This documents the state in the CI output and may help debugging in case
of failure.

dergoegge commented:
ACK fa1b3abc834a90fb1cfbd5ac63deb28f3990c1fb
